app/cms/utils/constants.tsx

```diff
@@ -0,0 +1,11 @@
+export const STATUSPAGE_API_URL =
+  "https://api.statuspage.io/v1/pages/ytw5bdg6zr13/components"
+export const MAINNET_STATUSPAGE_ID = "fqvhhbc3hdw8"
+export const TESTNET_STATUSPAGE_ID = "g9d7vtywpdfq"
+export const CANARYNET_STATUSPAGE_ID = "s4z9n7p9pm3s"
+export const STATUSPAGE_API_KEY = "e6278eb7-27f1-4795-adce-9a0751a4a31c"
```

Comment:
@10thfloor not sure if you're ever planning to OSS this repo but if so you'll need to rotate these keys and move them into env vars unless these keys are public

Reply:
We will hopefully OSS one day.